.gitignore

```diff
@@ -45,6 +45,9 @@ typings/
 # Optional eslint cache
 .eslintcache
 
+# VSCode settings
+.vscode/
```

Comment:
I haven't typically seen this in individual projects `.gitignore` files.

Are you aware you can define a global `.gitignore` file on your machine? This might be better to include there.

https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore